content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,121 +1,90 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+Saving and sharing experiments is similar to [sharing regular project versions],
```

Comment:
This is better! This is still suboptimal intro though. The way I would read it - we are pushing commits (sorry, if we are going in circles). How can we explain this to people?

```
Aw we learned before, DVC experiments are utilizing Git internals to capture their metadata. As we mentioned, it's similar to Git commits but w/o actually polluting your Git repository history and everything is automated. This is great for one person iterating on the project, but what if we want to share this? If they are not Git commits, how do we make experiments history available?
```

can something like this help?

Reply:
I see what you mean ⌛

Reply:
OK @shcheklein . PTAL at the latest intro here (introduced in 675aacec).

> I didn't use the exact same ideas you suggested as to avoid reiterating on basic explanations of `dvc exp` or going into implementation details (which exist in the overview page)